sklearn/svm/tests/test_svm.py

```diff
@@ -224,9 +225,29 @@ def test_svr():
     # non-regression test; previously, BaseLibSVM would check that
     # len(np.unique(y)) < 2, which must only be done for SVC
     svm.SVR().fit(diabetes.data, np.ones(len(diabetes.data)))
+    svm.QuantileSVR().fit(diabetes.data, np.ones(len(diabetes.data)))
     svm.LinearSVR().fit(diabetes.data, np.ones(len(diabetes.data)))
 
 
+def test_quantilesvr():
+    # check that SVR with epsilon set to zero is equivalent to QuantileSVR where
+    # the quantile is set to 0.5
+
+    diabetes = datasets.load_diabetes()
+    qvr = svm.QuantileSVR(kernel="linear", C=1e3, quantile=0.5).fit(
+        diabetes.data, diabetes.target
+    )
+    score1 = qvr.score(diabetes.data, diabetes.target)
+
+    svr = svm.SVR(kernel="linear", C=1e3, epsilon=0.0).fit(
+        diabetes.data, diabetes.target
+    )
+    score2 = svr.score(diabetes.data, diabetes.target)
+
+    assert_allclose(np.linalg.norm(qvr.coef_), np.linalg.norm(svr.coef_), 1, 0.0001)
+    assert_almost_equal(score1, score2, 2)
+
+
```

Comment:
Could you please add tests that checks that it converges to the expected results (e.g. by measuring the pinball loss on the training set and checking that it's small) for different values of the quantile parameter, for instance on synthetic data in low dimension (e.g a 2D Gaussian blob).

Reply:
Okay, I will get to this soon! Thank you so much for reviewing!

Reply:
Alright, I added several additional tests (that I admit to have shamelessly stolen from `QuantileRegressor`. They test that the quantiles are well calibrated with any kernel, that illegal inputs raise appropriate errors, and, finally, that the results of (linear) QSVR give approximately the same result as a Nelder-Mead minimization of the pinball loss.